### PR TITLE
Docker container for Travis CI 

### PR DESCRIPTION
**Pull request by emanuelbuholzer**
Create a Dockerfile for creating a Docker image which is used for the Continuous Integration workflow.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    texlive-pictures \
+    pgf 
+    
+# Set the working directory to wd and copy the git repository into it
+WORKDIR /wd
+VOLUME ["/wd"]
+COPY . .
+
+# This is used to fix a bug saying gitinfo2.sty cannot be found
+ADD http://www.pirbot.com/mirrors/ctan/macros/latex/contrib/gitinfo2/gitinfo2.sty gitinfo2.sty
```
Comment (emanuelbuholzer) [inline Dockerfile]: This is a bit hacky if anyone knows a better way to install gitinfo2.sty, that'd be great!

**Comment by emanuelbuholzer** [comment]
Documentation is currently WIP.

**Comment by rzach** [comment]
Cool! Does this generate the docker image every time it runs or does docker cache these? Or do I have to do something extra to make that happen?

**Comment by emanuelbuholzer** [comment]
I haven't tested it in detail, but on first sight on Travis, id didn't rebuild the image every time. Although this was also the case for the build without Docker.

That's why I've written the comment on #149, as the Docker build is without an additional feature may be a bit obsolete.

**Pull request merged**